lib/avrora/storage/registry.ex

```diff
@@ -93,10 +93,26 @@ defmodule Avrora.Storage.Registry do
   @spec configured?() :: true | false
   def configured?, do: !is_nil(registry_url())
 
+  defp csr_references_lookup_function(response) do
```

Comment:
To comply with typespec of reference lookup function from Avrora.Schema we should always return a function with interface like

```elixir
@type reference_lookup_fun :: (String.t() -> {:ok, String.t()} | {:error, term()})
```

which means for us that we can re-use the default lookup function `Avrora.Schema.reference_lookup/1`